glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+## Application
+
+A process tree launched inside a [container](#container).
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
```

Comment:
On Wed, Aug 12, 2015 at 02:37:00PM -0700, W. Trevor King wrote:
> Allowing the default config filename to be overridden at runtime
> seems both useful and easy to implement…

For a concrete example, see [1], which supports both the IPFS gateway:

  # runc

and a shell for poking around:

  # runc config-sh.json

[1]: https://github.com/wking/oci-gentoo-minimal/tree/ipfs-gateway